apps/web/src/lib/posthog/react.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import { env } from '@orbitkit/env/web';
+import posthog from 'posthog-js';
+import { PostHogProvider } from 'posthog-js/react';
```

Comment:
I don't see a package.json update in this PR which installs `posthog-js` nor `posthog-node`? 🤔